### ISSUE TITLE
update-pot.py doesn't work with gettext-0.19.8

### ISSUE THREAD

**Issue by fedelibre** [frescobaldi/frescobaldi#1040]
I guess that it's due to a change in gettext. I'm using version 0.19.8.
Here's the error message:

```
$ cd frescobaldi_app/po
$ python update-pot.py 
msguniq: input file 'temp2.pot' doesn't contain a header entry with a charset specification
msgcat: error while opening "temp3.pot" for reading: No such file or directory
Traceback (most recent call last):
  File "update-pot.py", line 54, in <module>
    os.remove('temp3.pot')
OSError: [Errno 2] No such file or directory: 'temp3.pot'
```

So it seems that msguniq expects as input a pot file containing a header with a charset specification.
`md2pot.py` should be fixed to provide such a header.

**Comment by fedelibre** [comment]
I'm not sure what the header should be exactly, but something like this:

```
msgid ""
msgstr ""
"Content-Type: text/plain; charset=UTF-8\n"
```

but msguniq will treat it as a normal string and convert it to:

```
msgid ""
msgstr "Content-Type: text/plain; charset=UTF-8\n"
```

**Comment by fedelibre** [comment]
> I wonder why md2pot.py creates an UTF-8 encoded file. It shouldn't be needed for english manual:

There are a couple of files saved as UTF-8:

```
$ for i in userguide/*.md; do file $i; done | grep UTF
userguide/auto-completion.md: UTF-8 Unicode text
userguide/credits.md: UTF-8 Unicode text
```

Only credits.md contains needed UTF-8 characters, while auto-completion.md contains nice quotes: `“word”`. Should these be replaced with regular `""`?
@uliska What do you think?

**Comment by fedelibre** [comment]
IIUC, xgettext won't set the charset to UTF-8 for the program pot (temp1.pot in the update-pot.py script) unless there's a UTF-8 character in the source code (which is not the case).
However, this is not a problem, since msguniq will merge the headers of temp1.pot and temp2.pot (userguide pot) and use the header of the latter for the final temporary file (temp3.pot).

So what is needed is make sure that md2pot.py adds the following header to temp2.pot:

```
# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the frescobaldi package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: frescobaldi 3.0.0\n"
"Report-Msgid-Bugs-To: info@frescobaldi.org\n"
"POT-Creation-Date: 2018-08-23 15:52+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```

**Comment by fedelibre** [comment]
@wbsoft I think you fixed this issue with commit da8094ee663d562412df9469b4e6e675e0f6bb4b

**Comment by wbsoft** [comment]
> @wbsoft I think you fixed this issue with commit [da8094e](https://github.com/frescobaldi/frescobaldi/commit/da8094ee663d562412df9469b4e6e675e0f6bb4b)

I quess yes! :-)
